<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Conway's Game </string>
  </property>
  <widget class="QWidget" name="widget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>209</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>625</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_File"/>
    <addaction name="actionOpen_File"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Game"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionON_OFF_Impact"/>
    <addaction name="actionSet_XOR"/>
    <addaction name="actionSet_Replace"/>
   </widget>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionNext_Step"/>
    <addaction name="actionRun"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_author"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuGame"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_File"/>
   <addaction name="actionOpen_File"/>
   <addaction name="actionSave_Game"/>
   <addaction name="actionSettings"/>
   <addaction name="separator"/>
   <addaction name="actionNext_Step"/>
   <addaction name="actionRun"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionON_OFF_Impact"/>
   <addaction name="actionSet_XOR"/>
   <addaction name="actionSet_Replace"/>
   <addaction name="separator"/>
   <addaction name="actionClear"/>
   <addaction name="actionAbout_author"/>
  </widget>
  <action name="actionNew_File">
   <property name="icon">
    <iconset>
     <normalon>:/ConwayGameRes/newFileIcon.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>New Game</string>
   </property>
   <property name="toolTip">
    <string>Create a new game</string>
   </property>
   <property name="statusTip">
    <string>Create a new game</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="icon">
    <iconset>
     <normalon>:/ConwayGameRes/FileOpenIcon.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Open Game</string>
   </property>
   <property name="toolTip">
    <string>Upload saved game</string>
   </property>
   <property name="statusTip">
    <string>Upload saved game</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset>
     <normalon>:/ConwayGameRes/settingsIcon.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Edit Settings</string>
   </property>
   <property name="statusTip">
    <string>Edit Settings</string>
   </property>
  </action>
  <action name="actionNext_Step">
   <property name="icon">
    <iconset>
     <normalon>:/ConwayGameRes/nextStep.jpg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Next Step</string>
   </property>
   <property name="toolTip">
    <string>Next Step</string>
   </property>
   <property name="statusTip">
    <string>Next Step</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset>
     <normalon>:/ConwayGameRes/run.jpg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="toolTip">
    <string>Run</string>
   </property>
   <property name="statusTip">
    <string>Run</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset>
     <normalon>:/ConwayGameRes/clear.jpg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="toolTip">
    <string>Clear</string>
   </property>
   <property name="statusTip">
    <string>Clear Field</string>
   </property>
  </action>
  <action name="actionON_OFF_Impact">
   <property name="icon">
    <iconset>
     <normalon>:/ConwayGameRes/switcher.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>ON/OFF Impact</string>
   </property>
   <property name="toolTip">
    <string>ON/OFF Impact</string>
   </property>
   <property name="statusTip">
    <string>ON/OFF Impact</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset>
     <normalon>:/ConwayGameRes/stopIcon.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="statusTip">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionSet_XOR">
   <property name="icon">
    <iconset>
     <normalon>:/ConwayGameRes/xorMode.jpg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Set XOR</string>
   </property>
   <property name="statusTip">
    <string>Set XOR</string>
   </property>
  </action>
  <action name="actionSet_Replace">
   <property name="icon">
    <iconset>
     <normalon>:/ConwayGameRes/replaceMode.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Set Replace</string>
   </property>
   <property name="statusTip">
    <string>Set Replace</string>
   </property>
  </action>
  <action name="actionAbout_author">
   <property name="icon">
    <iconset>
     <normalon>:/ConwayGameRes/about.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>About author</string>
   </property>
   <property name="statusTip">
    <string>About author</string>
   </property>
  </action>
  <action name="actionSave_Game">
   <property name="icon">
    <iconset>
     <normalon>:/ConwayGameRes/saveIcon.jpg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Save Game</string>
   </property>
   <property name="statusTip">
    <string>Save Game</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
